src/riak_kv_qry_worker.erl

```diff
@@ -136,6 +136,10 @@ handle_info({{SubQId, QId}, {error, Reason} = Error},
 handle_info({{_SubQId, QId1}, _}, State = #state{qid = QId2}) when QId1 =/= QId2 ->
     %% catches late results or errors such getting results for invalid QIds.
     lager:debug("Bad query id ~p (expected ~p)", [QId1, QId2]),
+    {noreply, State};
+
+handle_info(_Unexpected, State) ->
+    lager:info("Not handling unexpected message \"~p\" in state ~p", [_Unexpected, State]),
```

Comment:
Formatting the unexpected message could result in a very large log message, for this gen_server in particular that handles subquery result chunks with an unbounded number of rows.